package.json

```diff
@@ -36,10 +36,14 @@
     "gulp-clean": "~0.3.1",
     "gulp-uglify": "~1.1.0",
     "gulp-shell": "~0.3.0",
-    "es6-promise": "~2.0.1"
+    "es6-promise": "~2.0.1",
+    "uglify-js": "~2.4.17"
   },
   "scripts": {
+    "all": "gulp all",
+    "build": "gulp build",
     "test": "gulp test",
+    "test:watch": "gulp test:watch",
```

Comment:
Oh this is `test:watch`. We will eventually have a general watch task for development I guess. When we had it we should hook it to `npm start`

BTW using `gulp` in your script means you're assuming user have gulp installed globally. We can make gulp a dev dependency and do `"test": "./node_moduels/..../gulp test`. Of course that won't work in Windows, but that can be solved too. But we can worry about Windows later (or never! 😂)